code/lib/cli/src/detect.ts

```diff
@@ -134,26 +133,8 @@ export function detectBuilder(packageManager: JsPackageManager, projectType: Pro
   }
 }
 
-export function isStorybookInstalled(
-  dependencies: Pick<PackageJson, 'devDependencies'> | false,
-  force?: boolean
-) {
-  if (!dependencies) {
-    return false;
-  }
-
-  if (!force && dependencies.devDependencies) {
-    if (
-      SUPPORTED_RENDERERS.reduce(
-        (storybookPresent, framework) =>
-          storybookPresent || !!dependencies.devDependencies[`@storybook/${framework}`],
-        false
-      )
-    ) {
-      return true;
-    }
-  }
-  return false;
+export function isStorybookInstatiated(configDir: string) {
```

Comment:
Maybe the function could use a default value if no custom configDir is passed?
```suggestion
export function isStorybookInstatiated(configDir = resolve(process.cwd(), '.storybook')) {
```

or

```js
export function isStorybookInstatiated(configDir: string) {
  const directory = configDir || resolve(process.cwd(), '.storybook')
```

Reply:
I agree.